.github/actions/terraform-deploy/action.yml

```diff
@@ -0,0 +1,62 @@
+name: Deploy
```

Comment:
this is already in https://github.com/NASA-IMPACT/veda-data-airflow/pull/111 so can we not add it here?

Reply:
yeah we can probably remove this one (it's also out of date).